sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/MesosResourcePool.java

```diff
@@ -11,30 +11,29 @@
 
 /**
  * A representation of the pool of resources available in a single {@link Offer}. Tracks the
- * consumption of the {@link Offer}'s resources, as they are matched with
- * {@link ResourceRequirement}s.
+ * consumption of the {@link Offer}'s resources.
  */
 public class MesosResourcePool {
     private static final Logger logger = LoggerFactory.getLogger(MesosResourcePool.class);
 
     private Offer offer;
     private Map<String, List<MesosResource>> unreservedAtomicPool;
-    private Map<String, Value> unreservedMergedPool;
-    private Map<String, MesosResource> reservedPool;
+    private Map<String, MesosResource> dynamicallyReservedPool;
```

Comment:
the different keying should probably be pointed out here:
- `dynamicallyReservedPoolByResourceId`?
- `reservableMergedPoolByRole`?